Packs/RedHat/ModelingRules/RedHatEventsCollectorXDRC/RedHatEventsCollectorXDRC.yml

```diff
@@ -0,0 +1,4 @@
+name: RedHatEventsCollectorXDRC
+id: RedHatEventsCollectorXDRC
+fromversion: 3.3.0
+rules: ''
```

Comment:
You do not have a schema, do you need it ?

Reply:
https://github.com/demisto/content/pull/19252/commits/71739f96eab4b8e30e32b39f1be422038d76a882